src/biguint/iter.rs

```diff
@@ -117,9 +117,9 @@ impl ExactSizeIterator for U32Digits<'_> {
 }
 
 #[cfg(not(u64_digit))]
-impl<'a> U32Digits<'a> {
+impl U32Digits<'_> {
     #[inline]
-    pub(super) fn new(data: &'a [u32]) -> Self {
+    pub(super) fn new(data: &[u32]) -> Self {
```

Comment:
This one doesn't work on 32-bit targets, I guess for the same reason that you couldn't make this change on `cfg(u64_digit)`.